Lib/test/test_subprocess.py

```diff
@@ -46,6 +46,12 @@
     SETBINARY = ''
 
 
+
+# bpo-31692: If COUNT_ALLOCS is defined, stderr is flooded with allocation
+# statistics and so cannot be checked in a reliable way
+skip_unless_usable_stderr = test_support.requires_type_collecting
```

Comment:
"types are immortal if COUNT_ALLOCS is defined" looks bad skip message for these tests.